src/main/java/cz/muni/airport/dao/IAirportDao.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package cz.muni.airport.dao;
+
+import cz.muni.airport.model.Airport;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ *
+ * @author Jiri Krejci
+ */
+public interface IAirportDao {
+    
+    	public void create(Airport airport) throws IllegalArgumentException;
```

Comment:
v interface nemusis davat public, chyba to nie je ale nie je to povinne, interface je vzdy cely public :) len do buducna aby si vedel